contrib/ansible/getting-started.md

```diff
@@ -0,0 +1,122 @@
+[certificates] Valid certificates and keys now exist in "/etc/kubernetes/pki"
+[kubeconfig] Wrote KubeConfig file to disk: "/etc/kubernetes/admin.conf"
+[kubeconfig] Wrote KubeConfig file to disk: "/etc/kubernetes/kubelet.conf"
+[kubeconfig] Wrote KubeConfig file to disk: "/etc/kubernetes/controller-manager.conf"
+[kubeconfig] Wrote KubeConfig file to disk: "/etc/kubernetes/scheduler.conf"
+[apiclient] Created API client, waiting for the control plane to become ready
+[apiclient] All control plane components are healthy after 42.002391 seconds
+[token] Using token: 43a25d.420ff2e06336e4c1
+[apiconfig] Created RBAC rules
+[addons] Applied essential addon: kube-proxy
+[addons] Applied essential addon: kube-dns
+
+Your Kubernetes master has initialized successfully!
+
+To start using your cluster, you need to run (as a regular user):
+
+  mkdir -p $HOME/.kube
+  sudo cp -i /etc/kubernetes/admin.conf $HOME/.kube/config
+  sudo chown $(id -u):$(id -g) $HOME/.kube/config
+
+You should now deploy a pod network to the cluster.
+Run "kubectl apply -f [podnetwork].yaml" with one of the options listed at:
+  http://kubernetes.io/docs/admin/addons/
+
+You can now join any number of machines by running the following on each node
+as root:
+
+  kubeadm join --token 43a25d.420ff2e06336e4c1 172.31.7.230:6443
+
+```
+## Step 3:
+Use kubeadm join to add each of the remaining nodes to your cluster. (Note: Uses token that was generated during cluster init.)
+```console
+$sudo kubeadm join --token 43a25d.420ff2e06336e4c1 172.31.7.230:6443 --skip-preflight-checks
+[kubeadm] WARNING: kubeadm is in beta, please do not use it for production clusters.
+[preflight] Skipping pre-flight checks
+[discovery] Trying to connect to API Server "172.31.7.230:6443"
+[discovery] Created cluster-info discovery client, requesting info from "https://172.31.7.230:6443"
+[discovery] Cluster info signature and contents are valid, will use API Server "https://172.31.7.230:6443"
+[discovery] Successfully established connection with API Server "172.31.7.230:6443"
+[bootstrap] Detected server version: v1.7.6
+[bootstrap] The server supports the Certificates API (certificates.k8s.io/v1beta1)
+[csr] Created API client to obtain unique certificate for this node, generating keys and certificate signing request
+[csr] Received signed certificate from the API server, generating KubeConfig...
+[kubeconfig] Wrote KubeConfig file to disk: "/etc/kubernetes/kubelet.conf"
+
+Node join complete:
+* Certificate signing request sent to master and response
+  received.
+* Kubelet informed of new secure connection details.
+
+Run 'kubectl get nodes' on the master to see this machine join.
+```
+At the end of Step 3 you should have a kubernetes cluster up and running and ready for deployment. 
+
+## Step 4:
+Please follow the instructions [here](https://kubernetes.io/docs/setup/independent/create-cluster-kubeadm/) to deploy CNI network plugins and start a demo app.
```

Comment:
Use this link: https://kubernetes.io/docs/setup/independent/create-cluster-kubeadm/#pod-network